_posts/fr/newsletters/2024-02-14-newsletter.md

```diff
@@ -0,0 +1,219 @@
+---
+title: 'Bulletin Hebdomadaire Bitcoin Optech #289'
+permalink: /fr/newsletters/2024/02/14/
+name: 2024-02-14-newsletter-fr
+slug: 2024-02-14-newsletter-fr
+type: newsletter
+layout: newsletter
+lang: fr
+---
+
+Le bulletin de cette semaine résume les idées d'amélioration du relai après le déploiement du mempool en cluster, décrit les résultats
+de recherches sur les topologies et les tailles des sorties d'ancrage de style LN en 2023, annonce un nouvel hôte pour la liste de
+diffusion Bitcoin-Dev et encourage les lecteurs à célébrer la Journée du logiciel libre en remerciant les contributeurs de logiciels
+libres. Nous incluons également nos sections habituelles résumant une réunion du Bitcoin Core PR Review Club et décrivant les changements notables apportés aux principaux logiciels d'infrastructure Bitcoin.
+
+## Nouvelles
+
+- **Idées d'amélioration du relais après le déploiement du mempool en cluster :**
+  Gregory Sanders [a posté][sanders future] sur Delving Bitcoin plusieurs
+  idées permettant aux transactions individuelles d'opter pour certaines politiques de mempool après la mise en place complète,
+  le test et le déploiement du [mempool en cluster][topic cluster mempool]. Ces améliorations s'appuient sur les fonctionnalités du
+  [relais de transaction v3][topic v3 transaction relay] en assouplissant certaines de ses règles qui pourraient ne plus être nécessaires
+  et en ajoutant l'exigence qu'une transaction (ou un [paquet][topic package relay] de transactions) paie un taux de frais qui les rendra
+  probablement extraites dans le prochain bloc ou les deux prochains blocs.
+
+- **Que se serait-il passé si les règles v3 avaient été appliquées aux sorties d'ancrage il y a un an ?**
+  Suhas Daftuar [a posté][daftuar retrospective] sur Delving Bitcoin à propos
+  de ses recherches sur l'idée d'appliquer automatiquement la politique de relais de transaction v3 aux transactions de confirmation
+  et de modification des frais de style LN [anchors-style][topic anchor outputs] (voir le [Bulletin #286][news286 imbued] pour la
+  proposition sous-jacente _imbued v3_). En résumé, il a enregistré 14 124 transactions en 2023 qui ressemblaient à des dépenses
+  d'ancrage. Parmi celles-ci :
+
+  - Environ 94% <!-- (14124 - 856) / 14124 --> auraient réussi selon les règles v3.
+
+  - Environ 2,1% <!-- 302/14124 --> avaient plus d'un parent (par exemple, des tentatives de dépenses [CPFP][topic cpfp]). Certains
```

Comment:
Misformatting on this bullet and Im not initially sure why
<img width="832" alt="Screenshot 2024-02-15 at 1 43 08 PM" src="https://github.com/bitcoinops/bitcoinops.github.io/assets/1615772/89888e6d-cf6e-4bd9-a2e0-53e456343d10">

Reply:
I have a problem with my local configuration and cannot view before publishing, it's frustrating because each time you need to correct my mistake, , I'm really sorry for this.

Reply:
Let me know what your local issue is and maybe I can give you my suggestion

Reply:
we already try to solve this problem some month ago, but I don't find any solution.
It's a problem with my configuration with WSL.

Reply:
Thanks for your purpose, but I don't want to take more of your time, you already do a lot for me